pontoon/settings/base.py

```diff
@@ -728,6 +728,18 @@ def _allowed_hosts():
     'vi-vn', 'wo-sn', 'xh-za', 'yo-ng', 'zh-cn', 'zh-hk', 'zh-tw', 'zu-za',
 ]
 
+# Bleach tags and attributes
+ALLOWED_TAGS = [
+    'a', 'abbr', 'acronym', 'b', 'blockquote', 'br', 'code', 'em', 'i',
+    'li', 'ol', 'p', 'strong', 'ul',
+]
```

Comment:
Why not simply extend whatever `bleach` decides is safe? Same for `ALLOWED_ATTRIBUTES`.

```python
ALLOWED_TAGS = list(bleach.ALLOWED_TAGS).extend(['br', 'p'])
```